<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QWidget" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Control</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Polygon parameters:</string>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX major points:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbMaxMajorPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbMinMajorPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN major points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX smooth points:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbMaxSmoothPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN smooth points:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbMinSmoothPoints">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX round quality:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspbMaxRoundQuality">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN round quality:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspbMinRoundQuality">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnGeneratePolygon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate polygon</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
